src/main/java/org/mqttbee/annotations/DoNotImplement.java

```diff
@@ -1,3 +1,22 @@
+/*
+ *
+ * *
+ *  * Copyright 2018 The MQTT Bee project.
```

Comment:
'*' is part of the template and generated by IntelliJ.

Reply:
You're completely right. Please ignore the PR, I'll create a new one. Seems I completely missed that